ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/utils.ts

```diff
@@ -22,7 +22,7 @@ export const imageTypeLabelMap: Record<ImageType, string> = {
 
 export const cvesDiscoveredSinceLabelMap: Record<CVESDiscoveredSince, string> = {
     ALL_VULN: 'All time',
-    SINCE_LAST_REPORT: 'Last successful scheduled run report',
+    SINCE_LAST_REPORT: 'Last scheduled report that was successfully sent',
```

Comment:
@zhenpesky I thought the previous wording might not have been the clearest way to convey what we wanted to convey. This feels clearer to me. Let me know if it sounds better to you (cc: @dvail, @vjwilson )